release/changes.md

```diff
@@ -1,2 +1,2 @@
 - [NEW] Run tests against AGP 8.2.0-alpha16
-- [NEW] Run tests against AGP 8.1.1
```

Comment:
why is this showing a diff? is there a whitespace change?

Reply:
Yeah, I think it was without the newline at the end before. It's weird though, as I resolved it to the main branch variant of it when merging..